stretch_moveit_config/launch/moveit_rviz.launch.py

```diff
@@ -0,0 +1,49 @@
+from ament_index_python.packages import get_package_share_path
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.conditions import IfCondition, UnlessCondition
+from launch.substitutions import LaunchConfiguration
+
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    moveit_config_path = get_package_share_path('stretch_moveit_config')
+
+    ld = LaunchDescription()
+    ld.add_action(DeclareLaunchArgument('robot_description'))
+    ld.add_action(DeclareLaunchArgument('semantic_config'))
+
+    ld.add_action(DeclareLaunchArgument('debug', default_value='false', choices=['true', 'false']))
+    ld.add_action(DeclareLaunchArgument('rviz_config',
+                                        default_value=str(moveit_config_path / 'launch/moveit.rviz')))
+
+    planning_parameters = [str(moveit_config_path / 'config') + '/', LaunchConfiguration('pipeline'), '_planning.yaml']
+
+    rviz_parameters = [
+        {'robot_description': LaunchConfiguration('robot_description')},
+        {'robot_description_semantic': LaunchConfiguration('semantic_config')},
+        {'robot_description_planning': planning_parameters},
+        str(moveit_config_path / 'config/kinematics.yaml'),
+    ]
+
+    rviz_node = Node(package='rviz2', executable='rviz2',
```

Comment:
Ocassionally, Rviz doesn't show the interactive markers needed to move the joints around in order to set a target state for MoveIt2 to plan to. This error may be relevant:

```
[rviz2-4] Warning: class_loader.impl: SEVERE WARNING!!! A namespace collision has occured with plugin factory for class rviz_default_plugins::displays::InteractiveMarkerDisplay. New factory will OVERWRITE existing one. This situation occurs when libraries containing plugins are directly linked against an executable (the one running right now generating this message). Please separate plugins out into their own library or just don't link against the library and use either class_loader::ClassLoader/MultiLibraryClassLoader to open.
[rviz2-4]          at line 253 in /opt/ros/galactic/include/class_loader/class_loader_core.hpp
```

Reply:
May also be relevant?

```
[rviz2-4] [ERROR] [1634457207.943644000] [moveit_ros_visualization.motion_planning_frame]: Action server: /recognize_objects not available
```

Reply:
And also:

```
[rviz2-4] [ERROR] [1634457209.057952045] [moveit_dynamics_solver.dynamics_solver]: Group 'mobile_base_arm' does not have a parent link
[rviz2-4] [ERROR] [1634457209.057979049] [moveit_dynamics_solver.dynamics_solver]: Group 'position' does not have a parent link
```

Reply:
Full log available [here](https://gist.github.com/hello-binit/d10ce29c4a8d7250481267e3f824234e).